lib/ui/view/draft/app/data_holder_class.dart

```diff
@@ -0,0 +1,14 @@
+
+class Objects {
```

Comment:
Why is this class named ```Objects```?
also move this file to the models directory where all the model are and name it properly

Reply:
done